resources/tentative/complex-static-html/src/big-dom-generator/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { renderToStaticMarkup } from 'react-dom/server';
+import { App } from './app';
+import { genCss } from './gen-css'
+
+import './app.css';
+
+const fs = require('fs');
+
+// Generate css matching and non-matching complex css selectors targeted at the todoMVC items.
```

Comment:
This comment simply repeats what the code conveys.
I generally find these "what" comments distracting. Remove?